x-pack/plugins/cross_cluster_replication/public/app/sections/home/auto_follow_pattern_list/components/detail_panel/detail_panel.js

```diff
@@ -296,7 +297,12 @@ export class DetailPanel extends Component {
 
         <EuiSpacer size="l" />
 
-        <EuiLink href={indexManagementUri} data-test-subj="viewIndexManagementLink">
+        <EuiLink
+          href={routing._reactRouter.getUrlForApp('management', {
```

Comment:
For this, we should use reactRouteNavigate with parentHistory.You've already created task #67703 for tracking some files. I think to avoid page refresh we should change all getUrlForApp in management sections to 'reactRouteNavigate'  with parent history.

Reply:
I added this solution to the details panels. This help to avoid full-page refresh for these places.